storage/gcloud/aio/storage/storage.py

```diff
@@ -57,6 +57,50 @@ def __init__(self, *, service_file: Optional[Union[str, io.IOBase]] = None,
     def get_bucket(self, bucket_name: str) -> Bucket:
         return Bucket(self, bucket_name)
 
+    async def copy(self, bucket: str, object_name: str,
+                   destination_bucket: str, *, new_name: str = None,
+                   headers: dict = None, params: dict = None,
+                   timeout: int = 10,
+                   session: Optional[Session] = None) -> bytes:
+        token = await self.token.get()
+
```

Comment:
few nits:

- I like docstrings, everywhere (do you guys purposefully avoid them to promote self documenting code?) Here I'd put a docstring explaining the variables, the logic of rewriteto, and that link to gcloud docs
- I like like variable pairs like `bucket` and `destination_bucket` to have explicit pair name like `source_bucket` and `destination_bucket`

Reply:
The function signature is relatively straightforward and I'm following convention in this repo, but I could add the docstring to describe some of the function semantics.

Reply:
> do you guys purposefully avoid them to promote self documenting code?

No, but that's the best excuse for our laziness that I've heard yet :D That said -- yes, I would argue that docstrings are great but the more you can write code which doesn't need documentation, the better. One big problem with docstrings (and any docs) is how easy it is for them to fall out-of-sync with code; especially in situations like this where all the arguments are convention through the whole repo, having every function have the same docstring re: those copied values could quickly get out of hand!

Note that the above isn't meant to be an argument against writing docs/comments/docstrings, more of a warning of things to watch out for!